content/en/blog/_posts/2021-08-04-kubernetes-release-1.22.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: blog
+title: 'Kubernetes 1.22: XXX'
+date: 2021-08-04
+slug: kubernetes-1-22-release-announcement
+---
+
+**Authors:** [Kubernetes 1.22 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.22/release-team.md)
+
+We’re pleased to announce the release of Kubernetes 1.22, the second release of 2021!
+
+This release consists of 53 enhancements: 13 enhancements have graduated to stable, 24 enhancements are moving to beta, and 16 enhancements are entering alpha, and 3 features have been deprecated.
```

Comment:
Change "...are entering alpha, and 3 features..." to "...are entering alpha. Also, three features..." since the deprecations are not part of the 53 enhancements.

Reply:
Oh, and I changed "3" to "three" since numbers under 10 should be spelled.